index.html

```diff
@@ -1387,9 +1379,44 @@ <h4 class="accordion-title font-size-19">
                       >
                         <div class="accordion-content">
                           <p>
-                            Bitcoin is designed around the idea of using
-                            cryptography to control the creation and transfer of
-                            money in any time.
+                            GIT sessions
```

Comment:
Remove GIT sessions, Developer Series Sessions is enough

Reply:
resolved